modules/article/index.js

```diff
@@ -33,14 +33,21 @@ module.exports = {
         label: 'Basics',
         fields: [
           'title',
-          'visibility',
           'blurb'
         ]
       },
       main: {
         label: 'Content',
         fields: [
-          'main'
+          'main',
+          '_topics'
+        ]
+      },
+      utility: {
+        label: 'Utility',
+        fields: [
+          'slug',
+          'visibility'
```

Comment:
Are we doing this in core as well?

Reply:
we are .. does that mean we can leave this entire `utility` key out?

Reply:
Unless we're changing something in it, yes.